aws/PAS-network-environment-NAT.json

```diff
@@ -187,123 +197,221 @@
+                    "Ref": "PSMPMainSubnetCIDR"
+                },
+                "AvailabilityZone": {
+                    "Fn::Select": [
+                        "0",
+                        {
+                            "Fn::GetAZs": {
+                                "Ref": "AWS::Region"
+                            }
+                        }
+                    ]
+                },
+                "VpcId": {
+                    "Ref": "PASVPC"
+                },
                 "Tags": [
                     {
                         "Key": "Name",
-                        "Value": "Components to Vault peering"
+                        "Value": "PSMP Main"
                     }
-                ],
-                "VpcId": {
-                    "Ref": "ComponentsVPC"
-                },
-                "PeerVpcId": {
-                    "Ref": "VaultVPC"
-                }
+                ]
             }
         },
-        "ComponentsNACL": {
-            "Type": "AWS::EC2::NetworkAcl",
+        "PSMPSecondarySubnet": {
+            "Type": "AWS::EC2::Subnet",
             "Properties": {
+                "CidrBlock": {
+                    "Ref": "PSMPSecondarySubnetCIDR"
+                },
+                "AvailabilityZone": {
+                    "Fn::Select": [
+                        "1",
+                        {
+                            "Fn::GetAZs": {
+                                "Ref": "AWS::Region"
+                            }
+                        }
+                    ]
+                },
                 "VpcId": {
-                    "Ref": "ComponentsVPC"
-                }
+                    "Ref": "PASVPC"
+                },
+                "Tags": [
+                    {
+                        "Key": "Name",
+                        "Value": "PSMP Secondary"
+                    }
+                ]
             }
         },
-        "VaultNACL": {
-            "Type": "AWS::EC2::NetworkAcl",
+        "PTAMainSubnet": {
+            "Type": "AWS::EC2::Subnet",
             "Properties": {
+                "CidrBlock": {
+                    "Ref": "PTAMainSubnetCIDR"
+                },
+                "AvailabilityZone": {
+                    "Fn::Select": [
+                        "0",
+                        {
+                            "Fn::GetAZs": {
+                                "Ref": "AWS::Region"
+                            }
+                        }
+                    ]
+                },
                 "VpcId": {
-                    "Ref": "VaultVPC"
-                }
+                    "Ref": "PASVPC"
+                },
+                "Tags": [
+                    {
+                        "Key": "Name",
+                        "Value": "PTA Main"
+                    }
+                ]
             }
         },
-        "VaultPublicRT": {
-            "Type": "AWS::EC2::RouteTable",
+        "PTADRSubnet": {
```

Comment:
Is this comment still relevant?

Reply:
No.
PTA and CPM have Main-DR since they can't be with another CPM\PTA on the same resources (target accounts for CPM). Their configuration should be Main-DR and not Main-Secondary.